build.gradle.kts

```diff
@@ -101,8 +102,10 @@ val freemarkerVersion: String by project
 
 dependencies {
     implementation(gradleApi())
+    implementation(project(":pathsaver"))
     implementation("org.freemarker:freemarker:$freemarkerVersion")
     implementation(project(":kotlinx-knit-test"))
+    implementation("org.jetbrains.dokka:dokka-core:1.4.30")
```

Comment:
Spreading `1.4.30` version across multiple gradle files seems to be a potential dependency management issue. Please extract it to Gradle properties.

While looking onto it, I've found out that the project depends BOTH on dokka 0.10 and dokka 1.4.30.
Please stick only with 1.4.30